plugins/services/src/js/utils/InstanceUtil.js

```diff
@@ -38,13 +38,21 @@ const InstanceUtil = {
 
   getNode(task) {
     const nodesList = CompositeState.getNodesList();
-    const id = (task && task.agent && task.agent.id) || null;
+    let id = null;
+
+    if (task.agent && task.agent.id) {
+      id = task.agent.id;
+    } else if (task.agentId) {
```

Comment:
no need for `else if`. Seems like we have preferred to use only `if` whenever possible.

Reply:
also, please take a look at @nLight comments here https://github.com/dcos/dcos-ui/pull/2688/files/a4354273ccf7f4f58d4818b8ee7483675243fbcc#diff-419e17a15d4143340f20a92b4aa4fe76